onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2082,6 +2108,15 @@ ORT_API_STATUS_IMPL(OrtApis::CreateSessionFromArrayWithPrepackedWeightsContainer
   API_IMPL_END
 }
 
+
+ORT_API_STATUS_IMPL(OrtApis::GetTensorMemoryInfo, _In_ const OrtValue* value, _Outptr_ const OrtMemoryInfo** memory_info) {
+  TENSOR_READ_API_BEGIN
```

Comment:
>TENSOR_READ_API_BEGIN [](http://example.com/codeflow?start=2&length=21)

Shouldn't this be paired with a corresponding `TENSOR_READ_API_END`?

Reply:
Dont believe there is a corresponding end